src/main/java/com/redhat/red/build/finder/json/BuildConfig.java

```diff
@@ -0,0 +1,118 @@
+/**
+ * Copyright 2017 Red Hat, Inc.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.redhat.red.build.finder.json;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.redhat.red.build.finder.ConfigDefaults;
+import com.redhat.red.build.koji.model.xmlrpc.KojiChecksumType;
+
+import java.util.List;
+
+public class BuildConfig {
+    @JsonProperty("checksum-only")
+    private Boolean checksumOnly;
+
+    @JsonProperty("checksum-type")
+    private KojiChecksumType checksumType;
+
+    @JsonProperty("archive-types")
+    private List<String> archiveTypes;
+
+    private List<String> excludes;
```

Comment:
Is an annotation missing here?

Reply:
No, the annotation is only required if the name doesn't match.